docs/docs/installation/manual_deploy.md

```diff
@@ -281,8 +281,16 @@ run the file: _(sudo permission needed)_
 ```bash
 composer install
 ```
+### Change folder permission after dependencies installation
 
 
+```bash
+sudo chmod -R 775 /var/www/mdc/{{timestamp_dir}}/vender
+```
+<pre class="spec">
+sudo chmod -R 775 /var/www/mdc/{{timestamp_dir}}/vender
```

Comment:
We dont have a folder called vender ;)  
We have a folder called vendor.. so i guess this is a typo :)

Reply:
It was typo :)